packages/swingset-xsnap-supervisor/src/index.js

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs';
+import { bundlePaths } from './paths.js';
+
+export { supervisorBundleSHA256 } from '../dist/supervisor.bundle.sha256.js';
```

Comment:
Feeling uneasy about a static import of a build output.

Maybe another approach and an abuse of `fs.readFileSync` is warranted?

```js
export const supervisorBundleInfo = Object.freeze({
  get sha256() {
    return fs.readFileSync('../dist/supervisor.bundle.sha256', 'utf-8');
  }
});
```

Reply:
Hm, let's see, if someone imports this package without first `yarn build`ing it, they won't get an error until their code reaches for either the bundle or the hash, at which point `fs.readFileSync` will throw. I guess that's fine, and yeah I was uncertain about the import approach too.